javascript-arrays/main.js

```diff
@@ -0,0 +1,15 @@
+var colors = ['red', 'white', 'blue'];
+console.log(colors[0]);
+console.log(colors[1]);
+console.log(colors[2]);
+console.log('America is ' + colors[0] + ',' + colors[1] + ',' + 'and ' + colors[2] + '.');
+colors[2] = 'green';
+
+console.log('Mexico is ' + colors[0] + ',' + colors[1] + ',' + 'and ' + colors[2] + '.');
+console.log('The value of colors: ' + colors);
+
+var students = ['John', 'Mike', 'Dave', 'Steve'];
+var numberOfStudents = [students.length];
```

Comment:
This only works because concatenating an array converts it to a string, you dont want to wrap it in square brackets, it converts it to an array.